goreleaserlib/goreleaser_test.go

```diff
@@ -152,3 +155,38 @@ release:
 `
 	createFile(t, "goreleaser.yml", yaml)
 }
+
+func TestInitProject(t *testing.T) {
+	var filename = "test_goreleaser.yml"
+
+	defer func() {
+		if _, err := os.Stat(filename); !os.IsNotExist(err) {
+			if err != nil {
+				t.Fatal(err.Error())
+			}
+
+			if err := os.Remove(filename); err != nil {
+				t.Fatal(err.Error())
+			}
+		}
+	}()
+
+	if err := InitProject(filename); err != nil {
+		t.Errorf("exepcted InitProject() to run, but got %v", err.Error())
+	}
+
+	file, err := os.Open(filename)
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	out, err := ioutil.ReadAll(file)
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	config := config.Project{}
+	if err := yaml.Unmarshal(out, &config); err != nil {
+		t.Errorf("Not a valid config. %s", err.Error())
```

Comment:
can you change those `t.Errorf` and `t.Fatal` statements to use the `assertify` package as the other tests do?